README.md

```diff
@@ -1,170 +1,107 @@
 
-### Key stores
+## Getting started
 
-`debug.keystore` and `release.keystore` files need to be present in the `app` folder. Also the file
-`app/keystore.properties` is needed with the following inside (replace where needed):
+The first step is to include the Core library into your project as a Gradle compile dependency:
 
-```data
-store_password=STORE_PASSWORD
-key_alias=KEY_ALIAS
-key_password=KEY_PASSWORD
+```groovy
+compile 'com.xmartlabs.bigbang:core:0.1.11'
 ```
 
-If you need to create them, run the following in the `app` folder:
+Then, add jCenter to the buildscript repositories:
 
-```bash
-keytool -genkey -v -keystore debug.keystore -alias KEY_ALIAS -keyalg RSA -keysize 2048 -validity 10000
-keytool -genkey -v -keystore release.keystore -alias KEY_ALIAS -keyalg RSA -keysize 2048 -validity 10000
+```groovy
+buildscript {
+    repositories {
+        jcenter()
+    }
+}
 ```
 
-### Generic keys file
-
-For the following services, add the files `app/src/production/res/values/keys.xml` and
-`app/src/staging/res/values/keys.xml` (for each environment) with this empty content:
-
-```xml
-<?xml version="1.0" encoding="utf-8" ?>
-<resources>
-    <string name="SOME_SERVICE" translatable="false">SOME_SERVICE_TOKEN</string>
-</resources>
+Also, you'll need to add the following repositories:
+
+```groovy
+repositories {
+    flatDir {
+        dirs 'libs'
+    }
+    maven { url "https://jitpack.io" }
+    maven { url 'https://maven.fabric.io/public' }
+    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+}
 ```
 
-### Fabric
-
-In order to use Fabric, create a file `app/fabric.properties` with the content (replace with the
-right values):
-
-```data
-apiSecret=YOUR_BUILD_SECRET
-apiKey=YOUR_API_KEY
+Then, include any of the following compile dependencies to add an specific module to your app:
+```groovy
+compile 'com.xmartlabs.bigbang:dbflow:0.1.11'
+compile 'com.xmartlabs.bigbang:retrofit:0.1.11'
+compile 'com.xmartlabs.bigbang:ui:0.1.11'
 ```
 
-### GCM
-
-Place the staging and production `google-services.json` files in `app/src/staging` and
-`app/src/production` respectively. These are created using [a Google app]
-(https://developers.google.com/mobile/add?platform=android&cntapi=gcm).
-
-## Genymotion
-
-We recommend using [Genymotion](https://www.genymotion.com) emulator.
-
-## Useful plugins
-
-The plugins for Android Studio that we consider useful are:
-
-<dl>
-    <dt>.gitignore</dt>
-    <dd>For highlighting .gitignore files and finding unused paths in it.</dd>
-    
-    <dt>Android ButterKnife Zelezny</dt>
-    <dd>For automatically binding views with ButterKnife.</dd>
-    
-    <dt>Android Material Design Icon Generator</dt>
-    <dd>Used to add common icons to the projects. Icons come from the <a href="https://design.google.com/icons/">
-    Material Icons</a>.</dd>
+The `dbflow` module is database related, while the `retrofit` module is service related.
```

Comment:
"While the `dbflow` module is database related, the `retrofit` module is service related."